src/inference/src/dev/threading/cpu_map_scheduling.cpp

```diff
@@ -58,18 +58,24 @@ std::vector<std::vector<int>> apply_hyper_threading(bool& input_value,
     return result_table;
 }
 
-bool apply_cpu_pinning(bool& input_value,
-                       const bool input_changed,
-                       const int num_streams,
-                       const std::vector<std::vector<int>>& proc_type_table) {
+bool get_cpu_pinning(bool& input_value,
+                     const bool input_changed,
+                     const int num_streams,
+                     const threading::IStreamsExecutor::ThreadBindingType bind_type,
+                     const std::vector<std::vector<int>>& proc_type_table) {
     int result_value;
     int num_sockets = proc_type_table.size() > 1 ? proc_type_table.size() - 1 : 1;
     bool latency = num_streams <= num_sockets && num_streams > 0;
 
     if (proc_type_table[0][EFFICIENT_CORE_PROC] > 0) {
-        result_value = input_changed ? input_value : (latency ? false : true);
+        result_value =
+            input_changed
+                ? input_value
+                : ((latency || bind_type == threading::IStreamsExecutor::ThreadBindingType::NUMA) ? false : true);
```

Comment:
Need double check default pinning setting for "Ecore only" case with latency hint.